### ISSUE TITLE
Todo: rewrite generation script

### ISSUE THREAD

**Issue by jooooscha** [NixNeovim/NixNeovimPlugins#3]
The original has written generation the script in fennel.
I want to improve it; mainly its error handling.

To improve it, either rewrite in a sane language, or learn fennel.

**Comment by examosa** [comment]
Are you considering any languages in particular? I'd be interested in helping out with this effort.

**Comment by jooooscha** [comment]
> I'd be interested in helping out with this effort.

Nice, that would be great.

> Are you considering any languages in particular?

Not really. I guess using a script language is not the worst idea, so maybe python?

But, I am open to any other suggestions

**Comment by Stupremee** [comment]
Hey,

I'm actually working on a similar project, which was mainly intended for my personal NixOS configuration.
Basically, I'm writing a small Rust CLI for generating `buildVimPluginFrom2Nix` expressions from a list of repsitories because I was annoyed that nixpkgs-unstable is missing so many plugins and I didn't want to update whole input just because a few Vim plugins need an update.

Then I discovered this project and noticed that my CLI may fit for this. 
It's also much faster than the fennel script, and can generate the whole `manifest.txt` from this repository in about 20 seconds on my machine.

It's in a pretty early state because I began writing it only a few days ago, but maybe you're interested: https://github.com/Stupremee/vinix
